MEvoLibGUI/nextflowApp/models.py

```diff
@@ -13,15 +13,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from django.db import models
 from pathlib import Path
 
+from django.db import models

```

Comment:
Migrations will need to be done again, as most of these forms do not exist anymore